src/components/questions/AddQuestionForm.jsx

```diff
@@ -1,32 +1,81 @@
-import React from 'react';
+import React, {useState} from 'react';
 import styles from './questions.module.css';
 import {useProductContext} from '../../context/ProductContext.jsx';
+import axios from 'axios';
+import { API_KEY } from '../../../config/config.js';
```

Comment:
The server will handle the API Key with its requests to heroku. So I think we can remove this since we have a server thats functioning.